extensions/idp/pkg/command/server.go

```diff
@@ -102,3 +122,77 @@ func Server(cfg *config.Config) *cli.Command {
 		},
 	}
 }
+
+func ensureEncryptionSecretExists(path string) error {
```

Comment:
For other services, we have the certificate generation in ocis-pkg/crypto (https://github.com/owncloud/ocis/blob/master/ocis-pkg/crypto/gencert.go). Should we move this logic, too?

Reply:
I would leave it here for now and only move it to the package when we need that same logic somewhere else.